tests/postman/newman/collection.json

```diff
@@ -13576,6 +13576,267 @@
         }
       ]
     },
+    {
+      "name": "/v2/customers/:id",
```

Comment:
write a test for 1) get customer changes (group stuff) 2) adjust the user getting test for checking that there are no addresses returned

Reply:
okay